gk/bpf.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Gatekeeper - DoS protection system.
+ * Copyright (C) 2016 Digirati LTDA.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "gatekeeper_gk.h"
+#include "bpf.h"
+
+static const struct rte_bpf_xsym flow_handler_init_xsym[] = {
+};
+
+static const struct rte_bpf_xsym flow_handler_pkt_xsym[] = {
+};
+
+static int
+__bpf_jit_if_possible(struct rte_bpf *bpf, rte_bpf_jitted_func_t *ret_f,
+		unsigned int index, const char *name)
+{
+	struct rte_bpf_jit jit;
+	int rc;
+
+	rc = rte_bpf_get_jit(bpf, &jit);
```

Comment:
We can combine the definition and value assignment into one line: ``int rc = rte_bpf_get_jit(bpf, &jit);``